mentoris/settings.py

```diff
@@ -76,12 +76,18 @@
 # https://docs.djangoproject.com/en/4.1/ref/settings/#databases
 
 DATABASES = {
+
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': BASE_DIR / 'db.sqlite3',
-    }
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
```

Comment:
Even though this database doesn't reveal any sensitive information it's still a change from main we don't want. Let's just keep the sqlite database for now since Aaron put it there